src/main/java/org/jboss/forge/arquillian/command/AddContainerDependencyStep.java

```diff
@@ -96,18 +99,33 @@ public Result execute(UIExecutionContext context) throws Exception {
       Map<Object, Object> ctx = context.getUIContext().getAttributeMap();
       Container container = (Container)ctx.get(ContainerSetupWizard.CTX_CONTAINER);
       String version = (String)ctx.get(ContainerSetupWizard.CTX_CONTAINER_VERSION);
+      Project project = getSelectedProject(context);
+      ArquillianFacet arquillian = project.getFacet(ArquillianFacet.class);
+      ArquillianConfig config = arquillian.getConfig();
 
-      containerInstaller.installContainer(
-            getSelectedProject(context),
-            container,
-            version,
-            getVersionedDependenciesMap());
+      final String profileId = container.getProfileId();
+
+      if (container. isSupportedByChameleon(version)) {
+         dependencyManager.addChameleonDependency(project);
+         if (config.isContainerWithDefault("true")) {
+            config.addContainer(profileId);
+         } else {
+            config.addContainerWithAttribute(profileId, "default", "true");
+         }
```

Comment:
But it says - if container is default - add it, otherwise make it default and add it. Which can be (I assume) done by only using second call? 

As a side note: can we have all qualifiers defaults? What is the default behaviour when we have n containers set to default?